NitroxClient/GameLogic/Bases/Spawning/BasePiece/BaseBioReactorSpawnProcessor.cs

```diff
@@ -19,15 +19,32 @@ public class BaseBioReactorSpawnProcessor : BasePieceSpawnProcessor
             TechType.BaseBioReactor
```

Comment:
Not related, but we should probably add a processor for BaseNuclearReactor techtype, it's almost a duplicate class of BioReactor in subnautica's code

Reply:
This should be made in a future "sync reactors" PR